src/transformers/models/bort/convert_bort_original_gluonnlp_checkpoint_to_pytorch.py

```diff
@@ -278,11 +278,21 @@ def check_and_map_params(hf_param, gluon_param):
     # Compare output of both models
     tokenizer = RobertaTokenizer.from_pretrained("roberta-base")
 
-    input_ids = tokenizer.encode_plus(SAMPLE_TEXT)["input_ids"]
-
     # Get gluon output
+    input_ids = tokenizer.encode_plus(SAMPLE_TEXT, add_special_tokens=False)["input_ids"]
+
+    # tried out a bunch of different token its -> all unsucessful
+    #    real_mask_token_id = 3
+    real_mask_token_id = tokenizer.mask_token_id
```

Comment:
tried out all tokens from 0 - 10 as well but all yield the same bad results being the `<unk>` token